apps/wallet/src/ui/app/components/menu/content/index.tsx

```diff
@@ -52,17 +48,11 @@ function MenuContent() {
 			<ErrorBoundary>
 				<MainLocationContext.Provider value={mainLocation}>
 					<Routes location={menuUrl || ''}>
-						<Route path="/" element={<MenuList />} />
-						<Route path="/accounts" element={<AccountsSettings />}>
```

Comment:
Can we remove the `AccountsSettings` component as well? It seems it's not used any more